src/text-field-base.css.ts

```diff
@@ -72,16 +72,16 @@ export const textArea = style([
 ]);
 
 export const textAreaWithLabel = style({
-    marginTop: 28,
+    paddingTop: 28,
```

Comment:
Unrelated change. This was causing the upper area of a TextField to not be part of the text area if multiline is set to true